optimism/src/keccak/tests.rs

```diff
@@ -625,18 +618,28 @@ fn test_keccak_decomposable_folding() {
         let (_scheme, _final_constraint) =
             DecomposableFoldingScheme::<KeccakConfig>::new(constraints, vec![], &srs, domain, ());
 
+        /*
         // Fold Sponge(Absorb(Only))
-        {
```

Comment:
Note that these should not be in this PR. But, ok to merge to avoid blocking and wait another CI.